js/main.js

```diff
@@ -1,44 +1,58 @@
-function copyMatsList() { // copies matsList to clipboard
+function selectOption() {
+    let select = document.getElementById("dropdown-select");
+    let selected = select.value;
+    recipesViewer(selected);
+}
+
+function recipesViewer(selected) {
+    let recipe = '';
+    for (let value of crafting[selected]) {
+        recipe += `${ value.quantity }x ${ value.material } <br>`;
+    }
+    document.getElementById("recipebox").innerHTML = recipe;
+}
+
+function copyMatsList() {
     let elementAmount = collectAmount();
     let collection = calculateTotalMats(elementAmount);
     let matsList = toList(collection);
-    let copyList = document.createElement("textarea"); // creates textarea element
-    document.body.appendChild(copyList); // attaches textarea element to HTML body
+    let copyList = document.createElement("textarea");
+    document.body.appendChild(copyList);
     copyList.value = matsList;
     copyList.select();
     document.execCommand("copy"); // copies to clipboard
-    document.body.removeChild(copyList); // removes textarea from HTML body
+    document.body.removeChild(copyList);
 }
 
-function toList(collection) { // converts collection Object to string matsList
+function toList(collection) {
     let matsList = '';
     let displayList = '';
     for (let key of Object.keys(collection)) { // key = each key in collection Object
-        matsList += `${collection[key]}x ${key} \n`; // sets matsList formating for copy
-        displayList += `${collection[key]}x ${key} <br>`;
+        matsList += `${ collection[key] }x ${ key } \n`;
+        displayList += `${ collection[key] }x ${ key } <br>`;
     }
     document.getElementById("textbox").innerHTML = displayList;
     return matsList;
 }
 
-function calculateTotalMats(elementAmount) { // multiplies each key quantity against user input
+function calculateTotalMats(elementAmount) {
     let collection = {};
     for (let key of Object.keys(crafting)) { // key = key of crafting Object
         for (let value of crafting[key]) { // value = value associated with key from crafting Object
-            let userInput = elementAmount[key];
+            let userInput = elementAmount[key]; // userInput = each key in elementAmount
             if (userInput <= 0) {
                 continue;
             }
             if (!collection.hasOwnProperty(userInput)) {
                 collection[value.material] = 0;
             }
-            collection[value.material] += value.quantity * userInput; // multiplies each quantity against elementAmount
+            collection[value.material] += value.quantity * userInput; // multiplies each key quantity against user input

```

Comment:
Heh, this is sensitive to typing letters and then gives this:
![image](https://user-images.githubusercontent.com/35440982/120051601-41629200-bfef-11eb-8a71-02244fc46930.png)